backend/src/migrate-api-keys.ts

```diff
@@ -0,0 +1,29 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
just curious — this seems like a weird place to put this script, why don't we run it as a database migration and keep track of it with all the other migration scripts there?

Reply:
Cos it's not a schema migration, there's no `down`, it's a one-way-street kinda activity